addons/website/views/website_visitor_views.xml

```diff
@@ -255,7 +255,6 @@
                 <field name="lang_id"/>
                 <field name="country_id"/>
                 <field name="visit_count"/>
-                <field name="page_ids"/>
```

Comment:
Si if you keep the field in the view it keeps crashing even though you added a "groups" on the field definition?
Would be nice to keep it in the view since this could cause issues with xpathing on stable.

Reply:
Is there a lot of xpath on search views? But anyway it shouldn't be needed to change this as the field is removed if the user is not in the right group.

Reply:
well this field is not searchable. But AWA got a point. We should keep it for xpathing (no xpath in our side though)